tests/consensus_tests/test_collection_update_not_blocked_by_busy_worker.py

```
@@ -0,0 +1,130 @@
+    try:
+        r = requests.patch(
+            f"{peer_uri}/collections/{COLLECTION}",
+            json={"optimizers_config": {"default_segment_number": 2}},
+            timeout=PATCH_CLIENT_TIMEOUT_SEC,
+        )
+    except requests.exceptions.Timeout:
+        elapsed = time.time() - start
+        raise AssertionError(
+            f"Collection update did not return within {PATCH_CLIENT_TIMEOUT_SEC}s "
+            f"(waited {elapsed:.1f}s) - consensus apply was blocked by the busy update worker"
+        )
```

Ruff (0.15.13)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)